### ISSUE TITLE
fail to attack

### ISSUE THREAD

**Issue by ybsu** [ronny3050/AdvFaces#3]
dear sir , thanks to your code, I have run the project successfully, but the images generated by the generator which is trained by myself cannot attack the victim images. I use target mode, what is the best weights for gan loss、 perturbation loss、 identity loss and pixel loss?    is there anything I should pay attention to?   Thank you sir !

**Comment by ronny3050** [comment]
Hi! 

Did you try the pre-trained model? The pre-trained model is only for obfuscation.
If you are training from scratch, please let me know what parameters you use (the loss weights) and maybe some training curves.
